ios/testmanagerd/xcuitestrunner.go

```diff
@@ -234,9 +234,9 @@ func RunXCUITest(bundleID string, testRunnerBundleID string, xctestConfigName st
 	if err != nil {
 		return make([]TestSuite, 0), fmt.Errorf("RunXCUITest: cannot browse user apps: %w", err)
 	}
-	info, err := getAppInfos(bundleID, testRunnerBundleID, apps)
+	info, err := getTargetAppInfo(bundleID, apps)
 	if err != nil {
-		return make([]TestSuite, 0), fmt.Errorf("RunXCUITest: cannot get app information: %w", err)
+		bundleID = ""
```

Comment:
shouldn't this be the case only if `bundleID` was already empty? If we provide a bundle id and this fails that's an error, isn't it?

Reply:
True. Altered the `if{}` block to only fetch data when needed.